teacher_dashboard/student-list.php

```diff
@@ -0,0 +1,171 @@
+<?php
+session_start();
+//echo strlen($_SESSION['instructor']); 
+// if (strlen($_SESSION['instructor'])==0) {
+// 	echo "entered if";
+// 	header('location:logout.php');
+// } 
+
+?>
+<!DOCTYPE HTML>
+<html>
+<head>
+<title>Students Details</title>
+
+<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+<!-- Bootstrap Core CSS -->
+<link href="css/bootstrap.css" rel='stylesheet' type='text/css' />
+<!-- Custom CSS -->
+<link href="css/style.css" rel='stylesheet' type='text/css' />
+<!-- font CSS -->
+<!-- font-awesome icons -->
+<link href="css/font-awesome.css" rel="stylesheet"> 
+<!-- //font-awesome icons -->
+ <!-- js-->
+<script src="js/jquery-1.11.1.min.js"></script>
+<script src="js/modernizr.custom.js"></script>
+<!--webfonts-->
+<link href='//fonts.googleapis.com/css?family=Roboto+Condensed:400,300,300italic,400italic,700,700italic' rel='stylesheet' type='text/css'>
+<!--//webfonts--> 
+<!--animate-->
+<link href="css/animate.css" rel="stylesheet" type="text/css" media="all">
+<script src="js/wow.min.js"></script>
+	<script>
+		 new WOW().init();
+	</script>
+<!--//end-animate-->
+<!-- chart -->
+<script src="js/Chart.js"></script>
+<!-- //chart -->
+<!--Calender-->
+<link rel="stylesheet" href="css/clndr.css" type="text/css" />
+<script src="js/underscore-min.js" type="text/javascript"></script>
+<script src= "js/moment-2.2.1.js" type="text/javascript"></script>
+<script src="js/clndr.js" type="text/javascript"></script>
+<script src="js/site.js" type="text/javascript"></script>
+<!--End Calender-->
+<!-- Metis Menu -->
+<script src="js/metisMenu.min.js"></script>
+<script src="js/custom.js"></script>
+<link href="css/custom.css" rel="stylesheet">
+<!--//Metis Menu -->
+
+<style>
+.main-page{
+	position: relative;
+	top: 0px;
+	border: 2px solid;
+	height: 70%;
+	overflow-y: auto;
+}
+</style>
+
+
+</head> 
+<body class="cbp-spmenu-push">
+<div class="main-content">
+		
+	<?php include_once('includes/sidebar.php');?>
+		
+	<?php include_once('includes/header.php');?>
+		<!-- main content start-->
+		<div id="page-wrapper" class="row calender widget-shadow">
+			<div class="main-page">
+	
+			
+	<?php
+include "includes/dbconnection.php";
+$sql = "SELECT * FROM joined_courses";  

```

Comment:
this query will fetch all students enrolled in all the courses, we wanted  students list for individual courses..."where" clause is missing